frontend/src/APIClients/queries/StoryQueries.ts

```diff
@@ -126,7 +128,7 @@ export const buildCommentsQuery = (
   resolved: boolean | null,
 ): QueryInformation => {
   let queryString = gql`
-    query ComentsByStoryTranslation {
+    query CommentsByStoryTranslation {
```

Comment:
🤦 i need to go back to school for english
this is my bad as well but - can you move this to a new file `CommentQueries.ts`?